cg/cli/set/case.py

```diff
@@ -77,14 +78,15 @@ def set_case(
     if panel_abbreviations:
         update_panels(case=case, panel_abbreviations=panel_abbreviations, status_db=status_db)
 
-    if priority:
+    if isinstance(priority, Priority):
         update_priority(case=case, priority=priority)
 
     status_db.session.commit()
 
 
-def abort_on_empty_options(options: list[str]) -> None:
-    if not any(options):
+def abort_on_empty_options(options: list[str], priority: Priority | None) -> None:
+    """Abort if options are empty and bypass_check for priority returns False"""
+    if not any(options) and not priority == Priority.research:
```

Comment:
Works potentially but is it not more a `if all(option is None for option in options)` we want?

Reply:
No, there is an empty tuple in there, it wont work